test/plugins/rule/handle-enter-in-example.js

```diff
@@ -0,0 +1,40 @@
+/** @jsx h */
+
+export default editor => {
+    editor.run('onKeyDown', { key: 'Enter' })
+    editor.run('onKeyDown', { key: 'Enter' })
+    editor.moveToEndOfBlock()
+    editor.run('onKeyDown', { key: 'Enter' })
+    editor.run('onKeyDown', { key: 'Enter' })
+    editor.run('onKeyDown', { key: 'Enter' })
+}
+
+export const input = <value>
+    <document>
+        <rule type="rule">
+            <statement>
+                <p>Statement</p>
+            </statement>
+            <ruleexample>
+                <p>Ex<cursor/>ample</p>
+            </ruleexample>
+        </rule>
+    </document>
+</value>
+
+export const output = <value>
+    <document>
+        <rule type="rule">
+            <statement>
+                <p>Statement</p>
+            </statement>
+            <ruleexample>
+                <p>Ex</p>
+            </ruleexample>
+            <ruleexample>
+                <p>ample</p>
+            </ruleexample>
+        </rule>
+        <p><text><cursor/></text></p>
```

Comment:
`<cursor/>` only does anything when placed in a text node, and slate-hyperscript doesn't insert any by itself, hence why `<text>` here is necessary.

Reply:
It is working without `text` and it is written as so in other tests

Reply:
Interesting, this must have been changed since I wrote tests for examples on which those are based (they still wrap cursor in text if the node would otherwise be empty).